e2e/main.sh

```diff
@@ -107,6 +107,30 @@ apiServer:
   subjectAltNames:
   - '"${USHIFT_IP}"'
 EOF' &>"${output_dir}/0001-setup.log"
+
+    ssh_cmd 'cat <<EOF | sudo tee /etc/microshift/lvmd.yaml
+    # The fields here reflect LVM state of the CI smoke-testing instance.
+    device-classes:
+      # The name of a device-class
+      - name: default
+
+        # The group where this device-class creates the logical volumes
+        volume-group: rhel
+
+        # Storage capacity in GiB to be spared
+        spare-gb: 0
+
+        # A flag to indicate that this device-class is used by default
+        default: true
+      - name: thin
+        volume-group: rhel
+        spare-gb: 0
+        default: false
+        type: thin
+        thin-pool:
+          name: thin
+          overprovision-ratio: 10.0
+    EOF' &>>"${output_dir}/0001-setup.log"
```

Comment:
I think EOF shouldn't have any leading whitespace

From sos:
```
Jun 27 16:13:18 i-062cfa4ac615c31ab.us-east-2.compute.internal microshift[37246]: infrastructure-services-manager I0627 16:13:18.520422   37246 apps.go:94] Applying apps api components/csi-snapshot-controller/csi_controller_deployment.yaml
Jun 27 16:13:18 i-062cfa4ac615c31ab.us-east-2.compute.internal microshift[37246]: kube-apiserver E0627 16:13:18.528440   37246 patch_podspecextractor.go:100] "failed to mutate object for PSA using SCC" err="pods \"pod-for-container-named-csi-snapshot-controller\" is forbidden: unable to validate against any security context constraint: [provider \"anyuid\": Forbidden: not usable by user or serviceaccount, provider restricted
-v2: .containers[0].runAsUser: Invalid value: 65534: must be in the ranges: [1000030000, 1000039999], provider \"restricted\": Forbidden: not usable by user or serviceaccount, provider \"nonroot-v2\": Forbidden: not usable by user or serviceaccount, provider \"nonroot\": Forbidden: not usable by user or serviceaccount, provider \"hostmount-anyuid\": Forbidden: not usable by user or serviceaccount, provider \"hostnetwork-v2\"
: Forbidden: not usable by user or serviceaccount, provider \"hostnetwork\": Forbidden: not usable by user or serviceaccount, provider \"hostaccess\": Forbidden: not usable by user or serviceaccount, provider \"privileged\": Forbidden: not usable by user or serviceaccount]"
Jun 27 16:13:18 i-062cfa4ac615c31ab.us-east-2.compute.internal microshift[37246]: kube-apiserver E0627 16:13:18.528480   37246 patch_podspecextractor.go:101] failed to mutate object for PSA using SCC: pods "pod-for-container-named-csi-snapshot-controller" is forbidden: unable to validate against any security context constraint: [provider "anyuid": Forbidden: not usable by user or serviceaccount, provider restricted-v2: .cont
ainers[0].runAsUser: Invalid value: 65534: must be in the ranges: [1000030000, 1000039999], provider "restricted": Forbidden: not usable by user or serviceaccount, provider "nonroot-v2": Forbidden: not usable by user or serviceaccount, provider "nonroot": Forbidden: not usable by user or serviceaccount, provider "hostmount-anyuid": Forbidden: not usable by user or serviceaccount, provider "hostnetwork-v2": Forbidden: not usa
ble by user or serviceaccount, provider "hostnetwork": Forbidden: not usable by user or serviceaccount, provider "hostaccess": Forbidden: not usable by user or serviceaccount, provider "privileged": Forbidden: not usable by user or serviceaccount]
Jun 27 16:13:18 i-062cfa4ac615c31ab.us-east-2.compute.internal microshift[37246]: infrastructure-services-manager I0627 16:13:18.530731   37246 recorder_logging.go:44] &Event{ObjectMeta:{dummy.176c9019430e2378  dummy    0 0001-01-01 00:00:00 +0000 UTC <nil> <nil> map[] map[] [] [] []},InvolvedObject:ObjectReference{Kind:Pod,Namespace:dummy,Name:dummy,UID:,APIVersion:v1,ResourceVersion:,FieldPath:,},Reason:DeploymentUpdated,M
essage:Updated Deployment.apps/csi-snapshot-controller -n kube-system because it changed,Source:EventSource{Component:,Host:,},FirstTimestamp:2023-06-27 16:13:18.530675576 +0000 UTC m=+48.523681613,LastTimestamp:2023-06-27 16:13:18.530675576 +0000 UTC m=+48.523681613,Count:1,Type:Normal,EventTime:0001-01-01 00:00:00 +0000 UTC,Series:nil,Action:,Related:nil,ReportingController:,ReportingInstance:,}
Jun 27 16:13:18 i-062cfa4ac615c31ab.us-east-2.compute.internal microshift[37246]: infrastructure-services-manager I0627 16:13:18.530808   37246 apps.go:94] Applying apps api components/csi-snapshot-controller/webhook_deployment.yaml
Jun 27 16:13:18 i-062cfa4ac615c31ab.us-east-2.compute.internal microshift[37246]: kube-apiserver E0627 16:13:18.538321   37246 patch_podspecextractor.go:100] "failed to mutate object for PSA using SCC" err="pods \"pod-for-container-named-csi-snapshot-webhook\" is forbidden: unable to validate against any security context constraint: [provider \"anyuid\": Forbidden: not usable by user or serviceaccount, provider restricted-v2
: .containers[0].runAsUser: Invalid value: 65534: must be in the ranges: [1000030000, 1000039999], provider \"restricted\": Forbidden: not usable by user or serviceaccount, provider \"nonroot-v2\": Forbidden: not usable by user or serviceaccount, provider \"nonroot\": Forbidden: not usable by user or serviceaccount, provider \"hostmount-anyuid\": Forbidden: not usable by user or serviceaccount, provider \"hostnetwork-v2\": F
orbidden: not usable by user or serviceaccount, provider \"hostnetwork\": Forbidden: not usable by user or serviceaccount, provider \"hostaccess\": Forbidden: not usable by user or serviceaccount, provider \"privileged\": Forbidden: not usable by user or serviceaccount]"
Jun 27 16:13:18 i-062cfa4ac615c31ab.us-east-2.compute.internal microshift[37246]: kube-apiserver E0627 16:13:18.538359   37246 patch_podspecextractor.go:101] failed to mutate object for PSA using SCC: pods "pod-for-container-named-csi-snapshot-webhook" is forbidden: unable to validate against any security context constraint: [provider "anyuid": Forbidden: not usable by user or serviceaccount, provider restricted-v2: .contain
ers[0].runAsUser: Invalid value: 65534: must be in the ranges: [1000030000, 1000039999], provider "restricted": Forbidden: not usable by user or serviceaccount, provider "nonroot-v2": Forbidden: not usable by user or serviceaccount, provider "nonroot": Forbidden: not usable by user or serviceaccount, provider "hostmount-anyuid": Forbidden: not usable by user or serviceaccount, provider "hostnetwork-v2": Forbidden: not usable
 by user or serviceaccount, provider "hostnetwork": Forbidden: not usable by user or serviceaccount, provider "hostaccess": Forbidden: not usable by user or serviceaccount, provider "privileged": Forbidden: not usable by user or serviceaccount]
Jun 27 16:13:18 i-062cfa4ac615c31ab.us-east-2.compute.internal microshift[37246]: infrastructure-services-manager I0627 16:13:18.540403   37246 recorder_logging.go:44] &Event{ObjectMeta:{dummy.176c901943a1cd04  dummy    0 0001-01-01 00:00:00 +0000 UTC <nil> <nil> map[] map[] [] [] []},InvolvedObject:ObjectReference{Kind:Pod,Namespace:dummy,Name:dummy,UID:,APIVersion:v1,ResourceVersion:,FieldPath:,},Reason:DeploymentUpdated,M
essage:Updated Deployment.apps/csi-snapshot-webhook -n kube-system because it changed,Source:EventSource{Component:,Host:,},FirstTimestamp:2023-06-27 16:13:18.540352772 +0000 UTC m=+48.533358777,LastTimestamp:2023-06-27 16:13:18.540352772 +0000 UTC m=+48.533358777,Count:1,Type:Normal,EventTime:0001-01-01 00:00:00 +0000 UTC,Series:nil,Action:,Related:nil,ReportingController:,ReportingInstance:,}
Jun 27 16:13:18 i-062cfa4ac615c31ab.us-east-2.compute.internal microshift[37246]: infrastructure-services-manager W0627 16:13:18.540893   37246 components.go:23] Failed to start csi plugin: unmarshalling lvmd file: error converting YAML to JSON: yaml: line 23: could not find expected ':'
Jun 27 16:13:18 i-062cfa4ac615c31ab.us-east-2.compute.internal microshift[37246]: infrastructure-services-manager E0627 16:13:18.540931   37246 manager.go:118] service infrastructure-services-manager exited with error: unmarshalling lvmd file: error converting YAML to JSON: yaml: line 23: could not find expected ':', stopping MicroShift
Jun 27 16:13:18 i-062cfa4ac615c31ab.us-east-2.compute.internal microshift[37246]: ??? I0627 16:13:18.541059   37246 run.go:219] Interrupt received
Jun 27 16:13:18 i-062cfa4ac615c31ab.us-east-2.compute.internal microshift[37246]: ??? I0627 16:13:18.541087   37246 run.go:224] Stopping services
```